CONTRIBUTING.md

```diff
@@ -100,6 +100,8 @@ Before you submit your pull request consider the following guidelines:
 
 That's it! Thank you for your contribution!
 
+**NOTE**: When you submit a documentation PR, you can skip the travis ci run just by adding `[ci skip]` to commit message.
```

Comment:
Would: `When submitting a documentation PR, you can skip the travis ci by adding `[ci skip]` to your commit message.` be better?